config/config.yml

```diff
@@ -55,6 +55,7 @@ production:
   subdomain: openstudios
 
 common:
+  bryant_street_studios_webhook_url: http://localhost:9292/webhook
```

Comment:
added the api key to `config.keys.yml` which is not checked in and managed in deployments.